packages/components/src/RecurringSelect/useScheduler.ts

```diff
@@ -0,0 +1,13 @@
+import { useState } from "react";
+import { DurationPeriod, SchedulerState } from "./types";
+
+export const useScheduler = (initialState: Partial<SchedulerState> = {}) => {
```

Comment:
this has been reduce down to a very small and simple function. does it still make sense to have as a hook? the `SchdulerState` is a single key now, `recurrence` and the `DurationPeriod` is the only other thing you really need, which can be imported on its own

also the name can be improved. it is no longer tied to the `Scheduler`. the name should reflect that